### ISSUE TITLE
Regarding Batch Files fo scheduling crawling jobs

### ISSUE THREAD

**Issue by Neel-Gagan** [dadoonet/fscrawler#710]
Kibana 6.4.2
Elasticsearch 6.4.2
FSCrawler 2.4
Deployed on Windows 10

I have different jobs for crawling different folders. is there any way i can make batch files and schedule these jobs using windows task schedular.

**Comment by Ganesh2409** [comment]
Yes, you can use batch file and schedule in task scheduler. Below format should work.
`bin\fscrawler --config_dir (location of the job settings file) jobname`
